tests/accel/upgrade/internal/upgradeparams/vars.go

```diff
@@ -0,0 +1,30 @@
+package upgradeparams
+
+import (
+	"fmt"
+
+	"github.com/openshift-kni/eco-gotests/tests/accel/internal/accelparams"
+	"github.com/openshift-kni/k8sreporter"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var (
+	// Labels represents the range of labels that can be used for test cases selection.
+	Labels = []string{accelparams.Label, Label}
+	// DeploymentName is the name of the test workload.
+	DeploymentName = "test-workload"
+	// TestNamespaceName is the namespace where the workload is deployed.
+	TestNamespaceName = "test-ns"
+	// ContainerLabelsMap labels in an map used when creating the workload container.
+	ContainerLabelsMap = map[string]string{"app": "test-workload"}
+	// ContainerLabelsStr labels in a str used when creating the workload container.
+	ContainerLabelsStr = fmt.Sprintf("%s=%s", "app", "test-workload")
+	// ServicePort is the workload service port.
+	ServicePort int32 = 8080
+	// ReporterNamespacesToDump tells to the reporter from where to collect logs.
+	ReporterNamespacesToDump = map[string]string{"accel-test": "accel-test"}
```

Comment:
Here you are trying to dump logs from non existing namespace.
Please change it to 

ReporterNamespacesToDump = map[string]string{"test-workload": "test-workload", "accel-upgrade-workload-ns":"accel-upgrade-workload-ns"}